synchronizer/synchronizer.go

```diff
@@ -71,8 +79,10 @@ func NewSynchronizer(
 		cancelCtx:          cancel,
 		ethTxManager:       ethTxManager,
 		zkEVMClient:        zkEVMClient,
+		eventLog:           eventLog,
```

Comment:
Do you init the envetLog variable? It looks like it is going to be nil at this point.

Reply:
The function `NewSynchronizer` set this field as the incomming parameter `eventLog *event.EventLog`...  unless the caller pass a nil the field is !=nil, if I missing something let me known